netci.groovy

```diff
@@ -0,0 +1,34 @@
+// Import the utility functionality.
+
+import jobs.generation.Utilities;
+import jobs.generation.InternalUtilities;
```

Comment:
Remove this line.  You only need the Utilities functionality since this is a public repro